### ISSUE TITLE
1.3.0-rc02 ReferenceError: darkColors$default is not defined in JS Canvas

### ISSUE THREAD

**Issue by eygraber** [JetBrains/compose-jb#2648]
Can repro using [my project](https://github.com/eygraber/portal/) and running `gradle :samples:kotlin-inject:js-app:jsBrowserDevelopmentRun`

Using Kotlin 1.8.0

```
ReferenceError: darkColors$default is not defined
    at MainView.Render$composable_djpyp0_k$ (MainView.kt?3944:38:16)
    at MainPortal.Render$composable_djpyp0_k$ (MainPortal.kt?53e1:36:14)
    at eval (main.js.kt?c052:23:18)
    at ComposableLambdaImpl.invoke_5qf8pc_k$ (ComposableLambda.js.kt?b351:116:22)
    at eval (portals-js-app.js:92:25)
    at eval (main.js.kt?c052:89:7)
    at ComposableLambdaImpl.invoke_6harzl_k$ (ComposableLambda.js.kt?b351:107:22)
    at eval (portals-js-app.js:129:25)
    at CompositionLocalProvider$composable (CompositionLocal.kt?a409:228:5)
    at eval (ComposeScene.skiko.kt?3a7b:371:13)
```

**Comment by eymar** [comment]
I check out the master branch in you repo (lastcommit - 3c283260d2e747a0f3a1e54d1d5d45b8ca056404)
And it runs successfuly (at least no error in the console)

![Screenshot 2023-01-19 113817](https://user-images.githubusercontent.com/7372778/213421127-1ef88597-11c3-4c1d-8657-48b9b6bffd31.png)


Perhaps, you may need to do a clean build.

**Comment by eygraber** [comment]
Looks like the issue was a dependency that was built using an older version of Kotlin/compose-jb - https://github.com/JetBrains/compose-jb/issues/2649

**Comment by eygraber** [comment]
@eymar maybe there's an underlying issue here with the build? I just had this happen again (in a different project, different error, but same idea) and had to clean, stop the daemon, and then run `jsBrowserDevelopmentRun --no-build-cache --no-configuration-cache --rerun-tasks` in order for it to work.

**Comment by eymar** [comment]
> @eymar maybe there's an underlying issue here with the build? I just had this happen again (in a different project, different error, but same idea) and had to clean, stop the daemon, and then run `jsBrowserDevelopmentRun --no-build-cache --no-configuration-cache --rerun-tasks` in order for it to work.

Was it right after update of kotlin version?

**Comment by eygraber** [comment]
In this latest case no. I was able to run the app fine yesterday after updating Kotlin, Gradle, compose-jb, etc...

Then I ran it again today and it was still fine. Then I made some changes to use the new DropdownMenu from material3 and got those weird errors. Then after I cleaned, killed the daemon, and ran `jsBrowserDevelopmentRun --no-build-cache --no-configuration-cache --rerun-tasks` and after that it worked (I tried without those flags as well and it still wasn't working).
